srcpkgs/redsocks/files/redsocks/run

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+_user=_redsocks
+pid_file=/run/redsocks.pid
+! [ -f $pid_file ] &&
+	touch $pid_file &&
+	chown $_user:$_user $pid_file
+
+exec 2>&1
+
+# Add $ARGS for more arguments
+[ -r ./conf ] && . ./conf
+: ${CONF_FILE:=/etc/redsocks.conf}
+# Make sure a configuration file exists
+[ -e "${CONF_FILE}" ] || exit 1
```

Comment:
This doesn't seem necessary to check for the existence of the file, redsocks will exit with an error message if the file doesn't exist, which is more helpful.

https://github.com/darkk/redsocks/blob/19b822e345f6a291f6cff6b168f1cfdfeeb2cd7d/main.c#L110